pkg/status/status-controller.go

```diff
@@ -381,7 +384,12 @@ func shouldSkipUpdate(old, new interface{}) bool {
 	return newMObj.GetResourceVersion() == oldMObj.GetResourceVersion()
 }
 
-func shouldSkipDelete(_ interface{}) bool {
+func objNotInThisWDS(obj interface{}, thisWDS string) bool {
+	if objWDS, ok := obj.(metav1.Object).GetLabels()[originWdsLabelKey]; ok {
```

Comment:
What applies this label?

Reply:
Oh, I see now that there are _two_ separate symbols for this label's name --- in different packages. Not the best arrangement.

Reply:
Yes I also noted that in the opening of #2455. If we want only one symbol, should the symbol be in pkg/transport/generic (generic_transport_controller.go), or in pkg/util (labels.go)?

Reply:
@waltforme, @pdettori, @nirrozenbaum: This label is part of the interface with the transport plugin. Since systems typically distinguish between their main interface with clients and their one or more interfaces with plugin, we could follow that pattern and establish a package for the parts of the interface with the transport plugin that have no better home. Or we could simply not maintain that level of organization (yet), and declare this label in the same `types.go` where the main interface is declared.